### PR TITLE
Install ptvsd and dependency packages

### PR DESCRIPTION
**Pull request by sushilkm**

### CODE REVIEW

File: {{cookiecutter.module_name}}/Dockerfile.arm32v7 [hunk 1]
```diff
@@ -29,6 +29,9 @@ RUN pip install --no-cache-dir \
     RPi.bme280 \
     RPi.GPIO
 
+RUN pip install --no-cache-dir setuptools wheel
+RUN pip install --no-cache-dir ptvsd==3.0.0
```
Comment (jessebenson) [inline {{cookiecutter.module_name}}/Dockerfile.arm32v7]: setuptools and wheel are already installed in Dockerfile.arm32v7.  Can you just add ptvsd==3.0.0 to the existing pip install, in sorted order?

File: {{cookiecutter.module_name}}/Dockerfile.amd64 [hunk 2]
```diff
@@ -9,6 +9,9 @@ RUN apt-get update && \
 RUN pip install --no-cache-dir \
     azure-iothub-device-client==1.4.0.0b0
 
+RUN pip install --no-cache-dir setuptools wheel
+RUN pip install --no-cache-dir ptvsd==3.0.0
```
Comment (jessebenson) [inline {{cookiecutter.module_name}}/Dockerfile.amd64]: Can you just put ptvsd==3.0.0 in the existing pip install, and add python-wheel and python-setuptools to the apt-get line

Comment (sushilkm) [reply {{cookiecutter.module_name}}/Dockerfile.amd64]: fixed

File: {{cookiecutter.module_name}}/Dockerfile.arm32v7 (hunk 1 above)
Comment (sushilkm) [reply {{cookiecutter.module_name}}/Dockerfile.arm32v7]: fixed

**Pull request merged**